impl/events.go

```diff
@@ -316,7 +316,9 @@ func (m *manager) OnChannelCompleted(chid datatransfer.ChannelID, success bool)
 	}
 	// send an error, but only if we haven't already errored for some reason
 	if chst.Status() != datatransfer.Failing && chst.Status() != datatransfer.Failed {
-		return m.channels.Error(chid, datatransfer.ErrIncomplete)
+		err := xerrors.Errorf("data transfer channel %s failed to transfer data: %w", completeErr)
```

Comment:
This seems to be missing an argument, we have %s and %w, but just completeErr as arg.

Reply:
Thanks, good catch 👍